### PR TITLE
S4t1

### PR DESCRIPTION
**Pull request by jesshamby**
Please can you review s4t1. Only file changed is index.php (others are from merging) :)

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+<body>
+    <nav class="navbar sticky-top justify-content-center">
+        <img width="150px" alt="Capysaurus Logo" src="Images/CapybarasaurusLogo.gif" />
+        <h1>Capynotasaurus</h1>
+    </nav>
+
+    <div class="col">
+        <div class="row ml-5">
+            <form class="d-flex flex-row" method="post" action="">
+                <input name="search" type="search" class="form-control m-2" id="search" placeholder="<?php echo $searchedValue; ?>">
```
Comment (EleanorRoberts) [inline index.php:61]: Would suggest 'value' instead of 'placeholder' so that you keep your own text when searched

File: index.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+                <input name="submit" type="submit" class="btn m-1" />
```
Comment (EleanorRoberts) [inline index.php]: Would suggest altering the hover colour, so it does not default to the background colour / appear transparent.

File: index.php [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+                <input id="reset" name="submit" type="submit" class="btn m-1" value="Clear" />
```
Comment (EleanorRoberts) [inline index.php]: Would suggest possibly adding aria label for accessibility for the submit/clear buttons.

**Review by EleanorRoberts — CHANGES_REQUESTED** [review]
Just a few comments around the buttons and accessibility :)

File: index.php (hunk 1 above)
Comment (willthornhill) [reply index.php:61]: Our clear button relies on the search input being empty. MVP

**Pull request merged**